code/src/main/java/com/onlinecontacttracing/storage/data-storage/PotentialContact.java

```diff
@@ -50,6 +50,22 @@ public int hashCode() {
     return emailOfPerson.hashCode(); 
   } 
 
+  /*
+   * Make class sortable
```

Comment:
Please describe what the sorting mechanism is.

Reply:
yep